package.json

```diff
@@ -48,39 +48,41 @@
     "compression": "^1.5.2",
     "cookie-session": "^1.2.0",
     "cors": "^2.7.1",
+    "datatype-expansion": "0.0.12",
     "debug": "^2.2.0",
     "finalhandler": "^0.4.0",
     "form-data": "^1.0.0-rc3",
     "http-errors": "^1.3.1",
     "invariant": "^2.1.0",
     "oauth2orize": "^1.0.1",
-    "osprey-method-handler": "^0.10.0",
+    "osprey-method-handler": "jstoiko/osprey-method-handler#raml1.0",
```

Comment:
Once the new version is released, remind to change this reference to the corresponding version (0.10.4 I guess from https://github.com/mulesoft-labs/osprey-method-handler/pull/18/files)